gradle.properties

```diff
@@ -14,3 +14,9 @@ androidKeystorePath=non-existant-keystore-dont-create-me.store
 androidKeystoreStorePassword=
 androidKeystoreKeyAlias=godtools
 androidKeystoreKeyPassword=
+
+# crashlytics beta signing config
+crashlyticsBetaKeystorePath=crashlytics/beta.keystore
+crashlyticsBetaKeystoreStorePassword=
+crashlyticsBetaKeystoreKeyAlias=godtools
+crashlyticsBetaKeystoreKeyPassword=
```

Comment:
The echo commands above will fill this value in.. or overwrite the key/value? My initial guess is that the echo would add extra lines.

Reply:
neither, it actually writes to ~/.gradle/gradle.properties

settings in this file override settings in local projects

Reply:
the blank lines are here to allow the build script to compile without missing properties

Reply:
got it, thanks